StRoot/StBFChain/BigFullChain.h

```diff
@@ -1712,6 +1714,13 @@ Bfc_st BFC[] = { // standard chains
    "StFmsPointMaker","StFmsPointMaker","Fill FMS clusters and points",                              kFALSE},
   {"fmsfps"   ,"","", "event,fmsDb",
    "StFmsFpsMaker","StFmsFpsMaker","Fill FPS association in FMS points",                            kFALSE},
+  // RHICf
+  {"rhicfDat" ,"","", "StEvent,RhicfUtil,rhicfdb",
+   "StRHICfRawHitMaker","StRHICfRawHitMaker","Fill RHICf raw data", kFALSE},
+  {"rhicfCali" ,"","", "StEvent,RhicfUtil,rhicfdb",
+   "StRHICfHitMaker","StRHICfHitMaker","RHICf calibration", kFALSE},
+  {"rhicfRec" ,"","", "StEvent,RhicfUtil,rhicfdb",
+   "StRHICfPointMaker","StRHICfPointMaker","RHICf reconstruction", kFALSE},
```

Comment:
I don't see codes for StRHICfHitMaker and StRHICfPointMaker. Am I missing something? Are those coming later? If so, I think the chain options should be introduced when those are introduced.

Reply:
Yes, StRHICfHitMaker and StRHICfPointMaker will be added later. (But these makers have already been made)
I will make new Pull-requests for calibration and reconstruction makers, including these BFC options later.